Makefile

```
@@ -61,15 +61,15 @@ $(SSH_KEY_PRIV_PATH): $(SSH_KEY_DIR)
 
 $(SSH_KEY_PUB_PATH): $(SSH_KEY_PRIV_PATH)
 
-.PHONY: gather checkenv clean destroy-libvirt start-vm network ssh bake wait-for-install-complete $(IMAGE_PATH_SNO_IN_LIBVIRT) $(NET_CONFIG) $(CONFIG_DIR)
+.PHONY: gather checkenv clean destroy-libvirt start-vm network ssh bake wait-for-install-complete $(IMAGE_PATH_SNO_IN_LIBVIRT) $(NET_CONFIG) $(CONFIG_DIR) help
 
 .SILENT: destroy-libvirt
 
 ### Install SNO from ISO
 start-iso: bootstrap-in-place-poc
 	make -C $(SNO_DIR) $@
 
-start-iso-abi: bootstrap-in-place-poc machineConfigs/internal-ip.yaml
+start-iso-abi: bootstrap-in-place-poc machineConfigs/internal-ip.yaml ## Install SNO cluster with bootstrap-in-place
```

```suggestion
start-iso-abi: bootstrap-in-place-poc machineConfigs/internal-ip.yaml ## Install SNO cluster with ABI
```

BIP is different than ABI